OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -540,5 +629,47 @@ string GetSourceDisplayName(IReadOnlyPackage source)
 
 			return name;
 		}
+
+		// Mute/UnMute code copied from MissionBrowserLogic.
+		float cachedMusicVolume;
+		void MuteSounds()
+		{
+			if (Game.Sound.MusicVolume > 0)
+			{
+				cachedMusicVolume = Game.Sound.MusicVolume;
+				Game.Sound.MusicVolume = 0;
+			}
+		}
+
+		void UnMuteSounds()
+		{
+			if (cachedMusicVolume > 0)
+				Game.Sound.MusicVolume = cachedMusicVolume;
+		}
+
+		void CleanLoadedAssets()
+		{
+			if (currentSound != null)
+				Game.Sound.StopSound(currentSound);
+
+			if (currentAudioStream != null)
+			{
+				currentAudioStream.Dispose();
+				currentAudioStream = null;
+			}
```

Comment:
You should be able to simplify this to
```suggestion
			currentAudioStream?.Dispose();
			currentAudioStream = null;
```